docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc

```diff
@@ -240,15 +240,15 @@ Create the Red Hat Container Registry token by following these steps:
 
 === Configuring your application to use a Red Hat Container Registry token 
 
-After creating the Red Hat Container Registry token, include the token in your application. When your application includes the Red Hat Container Registry token, you can access *registry.redhat.io* which improves your automation. 
+After creating and saving the Red Hat Container Registry token, you now need to configure it in your application's builds. When your workspace includes the Red Hat Container Registry token, you can access *registry.redhat.io* container images, enabling you to build off of trusted base images. 
 
 .Procedure  
 
 . Go to *Component Settings*. 
 . Select *Create new build secret*.
 . In the *Create new build secret* modal, add the *registry.redhat.io* token that you want to use as a pull secret:
 .. In the *Key* field, enter *.dockerconfigjson*.
-.. In the *Select or Enter Name* field, enter *registry-redhat.io*. 
+.. In the *Select or Enter Name* field, enter *registry-redhat-io*. 
 .. Enter the registry service account token and click *Create*.
 .. From your  command line, run the `oc secrets link appstudio-pipeline registry-redhat-io` command. 
```

Comment:
This line isn't clear to me either. You are in the RHTAP GUI and then you just tell people to go to the command line. Is there any documentation on configuring this that we can link to in order to help with the transition?

Reply:
We do not have any particular documentation on configuring this type of CLI work to help with the transition. 

We thought that providing some CLI instruction would help. I really like your additions, and I think that will help provide clarity and smooth out the transition from UI to CLI here.